scripts/k8conduit.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+# Connect to a backend service via an app instance
+#
+
+# TODO
+#
+# - usually more than one redis service. Set default and how to override?
+# - test against azure redis service when available
+# - confirm before running if interactive, a flag to run without confirmation?
+#
+
+Help()
```

Comment:
As discussed, can we use lowercase and hyphens for function names, as per https://google.github.io/styleguide/shellguide.html#s7.1-function-names

Reply:
done